nussknacker-dist/src/universal/conf/docker-application.conf

```diff
@@ -20,15 +20,15 @@ authentication: {
   redirectUri: ${?OAUTH2_REDIRECT_URI}
   accessTokenUri: ${?OAUTH2_ACCESS_TOKEN_URI}
   profileUri: ${?OAUTH2_PROFILE_URI}
-  authorizationHeader: ${?AUTHENTICATION_HEADER}
   accessTokenParams: {
-    grant_type: "authorization_code"
-  }
-  headers {
-    Accept: "application/json"
+    grant_type: ${?OAUTH2_GRANT_TYPE]
   }
   authorizeParams {
-    response_type: "code"
+    response_type: ${?OAUTH2_RESPONSE_TYPE}
+    scope: ${?OAUTH2_SCOPE}
+  }
+  headers {
```

Comment:
maybe we should have defaults here? i Accept, response_type etc.?

Reply:
Hmm, you are right. I will add default at docker.